src/CLI/clitree/cli-xml/mac.xml

```diff
@@ -320,5 +320,79 @@ limitations under the License.
     </ACTION>
     </COMMAND>
 
+<!--====================clear mac address-table dynamic <all | vlan | address>======================-->
+
+    <COMMAND
+        name="clear mac"
+        help="clear mac"
+        >
+    <ACTION>
+        builtin="clish_nop"
+    </ACTION>
+    </COMMAND>
+
+    <COMMAND
+        help="clear Mac address table"
+        name="clear mac address-table"
```

Comment:
Would prefer to have the command first and help next.

Reply:
done